src/util/compareThought.ts

```diff
@@ -5,10 +5,23 @@ import { EMOJI_REGEX } from '../constants'
 const regexPunctuation = /^[!@#$%^&*()\-_=+[\]{};:'"<>.,?\\/].*/
 const regexShortDateWithDash = /\d{1,2}-\d{1,2}/
 const regexShortDateWithSlash = /\d{1,2}\/\d{1,2}/
+const ignoredPrefixes = ['the ']
 
 /** Remove emojis and trailing/leading spaces from camparator inputs using regex. */
 const removeEmojisAndSpaces = (str: string) => str.replace(EMOJI_REGEX, '').trim()
 
+/** Remove ignored prefixes from comparator inputs. */
+const removeIgnoredPrefixes = (str: string) => {
+  const ignoredKeywords = ignoredPrefixes.join('|')
+  const regex = new RegExp(`^(${ignoredKeywords})(.*)`, 'gm')
```

Comment:
You can move the regex to the module scope since it does not change between invocations. Regex instantiation is not trivial like object literals.

Reply:
Yes, you are right. Also I have moved the ignoredPrefixes variable to `constants.ts`. Because one day you may need to access these ignored prefixes.